docs/conditional-artifacts-parameters.md

```diff
@@ -2,10 +2,8 @@
 
 > v3.1 and after
 
-The Conditional Artifacts and Parameters feature enables to assign the Step/DAG level artifacts or parameters based on
-expression. This introduces a new field `fromExpression: ...` under Step/DAG level output artifact and `expression: ...`
-under step/DAG level output parameter. Both use the
-[expr](https://github.com/antonmedv/expr/blob/master/docs/Language-Definition.md) syntax.
+The Conditional Artifacts and Parameters feature enables to assign the Step/DAG level artifacts or parameters based on an [expression](variables.md#expression).
```

Comment:
should we fix this old typo:
> ...feature enables to assign the Step/DAG level...

by saying:
>...feature enables you to assign the Step/DAG-level artifacts or parameters based on...

Reply:
Yea, good catch, I missed that. This whole paragraph could be reworded in [active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice) as well

Reply:
Rewrote the paragraph